### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by NatalieWolfe**
Massive simplification and refactoring of elastic search templates.

### CODE REVIEW

File: es.sh [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    remove_all_templates && remove_all_indexes
+}
+
+###################
+##               ##
+##    OPTIONS    ##
+##               ##
+###################
+
+while [[ $1 == -* ]]; do
```
Comment (zvxr) [inline es.sh]: neat

Comment (dkimdkimdkim) [inline es.sh]: concur

File: scripts/colors.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export red="\033[1;31m"
+export no_color="\033[0m"
+
+function cecho {
+    if $COLORS; then
+        echo -e $@
+    else
+        echo $@
+    fi
+}
```
Comment (dkimdkimdkim) [inline scripts/colors.sh]: nice.

File: templates/charts.json [hunk 3]
```diff
@@ -1,21 +1,13 @@
...
-    "order": 10,
-    "settings": {
-        "number_of_shards": 3,
-        "number_of_replicas": 1,
-        "index.refresh_interval": "1m",
-        "index.store.compress.stored": false,
-        "merge.policy.merge_factor": 10,
-        "replication": "async"
-    },
+    "template": "charts_*",
```
Comment (zvxr) [inline templates/charts.json]: I added a note on PLS-982 that recaching methods should be moved from PL repo to publisher.

Comment (NatalieWolfe) [inline templates/charts.json]: How does that relate to this template?

Comment (zvxr) [inline templates/charts.json]: method uses index alias -- won't work and should be removed/migrated as part of that ticket.

**Pull request merged**